Publisher/nl/database-import.md

```diff
@@ -1,116 +1,67 @@
 # Importeren van profielgegevens
-Er zijn verschillende manieren om nieuwe gegevens in te voeren of te importeren.
-Zo kun je profielen niet alleen met de hand toevoegen, maar kun je ze ook
-aanmaken of bewerken met de API, of importeren vanuit een CSV- of TXT-bestand
-(dit zijn bestanden die je onder meer in spreadsheetprogramma's kunt maken).
+Er zijn verschillende manieren waarop je nieuwe profielen binnen je database kunt toevoegen of wijzigen. Zo kun je profielen met de hand, via de [API](./apis), via [webformulieren](./webforms) of door middel van een import toevoegen, wijzigen of verwijderen. 
 
-Het importeren van CSV- of TXT-bestanden is krachtig en kan ook worden gebruikt
-voor gelaagde databases met collecties en subprofielen. Ook kun je periodieke
-imports maken die automatisch worden herhaald.
+De importfunctie in Copernica ondersteunt CSV- en TXT-bestanden. Je kunt deze bestandsformaten exporteren vanuit tekstverwerkingsprogramma's als Microsoft Word of spreadsheetprogramma’s als Excel.
```

Comment:
Is dit echt wat we willen communiceren? Word is misschien wel de slechtst denkbare manier om een adressenbestand in bij te houden.